src/components/PersonLink.tsx/PersonLink.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  parent: string | null
+  existingPerson: (value: string | null) => Person | undefined
+};
+
+export const PersonLink: React.FC<Props> = ({
+  parent,
+  existingPerson,
+}) => {
+  return (
+    <td>
+      {!parent
+        && '-' }
```

Comment:
```suggestion
}) => {
  if (!parent) {
    return (
     <td> - </td>
    );
  }
  
  return (
    <td>
```

would be a bit cleaner